manifests/config.pp

```diff
@@ -29,7 +29,7 @@
       }
 
       file {'/etc/tftpd.map':
-        content => template('tftp/tftpd.map'),
+        source  => 'puppet:///modules/tftp/tftpd.map',
         mode    => '0644',
         notify  => Class['xinetd'],
```

Comment:
<_<  Yeah I am trying to do too many things at the same time and didn't run my typical lint-via-vim.  Dooh!